apps/frontend/src/context/MockContext.tsx

```diff
@@ -8,19 +8,17 @@ declare global {
   }
 }
 
-const mockingOnFromEnvironment =
-  import.meta.env.VITE_USE_MOCK_DATA === 'true' ||
-  import.meta.env.VITE_USE_MOCK_DATA === 'TRUE' ||
-  false;
-
 const MockContext = React.createContext<{
   mocking: boolean;
-} | null>({
-  mocking: mockingOnFromEnvironment,
-});
+} | null>(null);
 
 export const MockProvider = ({ children }: { children: React.ReactNode }) => {
-  const [mocking, setMocking] = React.useState(false);
+  const mockingOnFromEnvironment =
+    import.meta.env.VITE_USE_MOCK_DATA === 'true' ||
+    import.meta.env.VITE_USE_MOCK_DATA === 'TRUE' ||
```

Comment:
Isn't this something that can be interpreted as a boolean though? 🤔

Reply:
It is possible, but then you would get VITE_USE_MOCK_DATA=FALSE, to still use mocks.
Not a big problem, but i guess i like the more explicit version. But the ? : version looks a bit better